frontend/cypress/integration/todo.spec.ts

```diff
@@ -2,7 +2,7 @@ describe('todo', () =>
 {
 	beforeEach(() =>
 	{
-		cy.visit('localhost:3000');
+		cy.visit('https://todos-dev.vercel.app');
```

Comment:
environment.baseUrl instead... extend all the env files

Reply:
Cypress is not reading the environmental files, I've changed baseUrl to hardcoded link on which cypress is working, and he is not responding